can't build _pack.c - SDK 7.0 doesn't include stdint.h

I can't build dulwich anymore on Windows 7 x64. I get this error using Windows SDK 7.0 (MSVC9).

> "_pack.c(21) fatal error C1083: Cannot open include file: 'stdint.h': No such file or directory"

I tried using this version
  http://msinttypes.googlecode.com/svn/trunk/stdint.h.
by copying "stdint.h" into dulwich folder, and I also tried compiling with sdk 7.1 (MSVC10) which includes stdint.h. Dulwich compiles without errors for both cases, and most of the tests pass, except for test_pack.py, which errors in TestThinPack, and almost all of the unittests in tests/compat fail.

The failures are all either the file is in use by another process (ThinPack), access denied (dulwich.tests.compat.test_client.DulwichMockSSHClientTest), or timed out (dulwich.tests.compat.test_client.DulwichTCPClientTest).

I think it's built fine, but I'm not 100% Are the tests supposed to fail, or are there some tests that I can run to see if _pack.pyd as built is functioning correctly? Thanks!

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -2,6 +2,7 @@ include NEWS
 include README
 include Makefile
 include COPYING
+include stdint.h
 recursive-include docs conf.py *.txt Makefile make.bat
 recursive-include examples *.py
 graft dulwich/tests/data

